code/game/objects/items/crab17.dm

```diff
@@ -30,6 +30,13 @@
 			var/datum/bank_account/B = i
 			B.being_dumped = TRUE
 		new /obj/effect/dumpeet_target(targetturf, L)
+
+		to_chat(user, span_notice("You have activated Protocol CRAB-17."))
+		message_admins("[ADMIN_LOOKUPFLW(user)] has activated Protocol CRAB-17.")
+		log_admin("[key_name(user)] has activated Protocol CRAB-17.")
```

Comment:
`log_admin` is used for logging admin actions (things like ASAY) so this line shouldn't be here.

Reply:
Glad you pointed it out since I wasn't sure the difference.  Fixed with https://github.com/tgstation/tgstation/pull/69522/commits/78f2d2fd32cf46227169895a231137a1939c47b9